modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/custom_range_facet.js

```diff
@@ -30,12 +30,12 @@ AUI.add(
 			instance.toInputName = config.toInputName;
 
 			if (instance.aggregationType === AGGREGATION_TYPES.RANGE) {
-				instance.fromInputPicker = document.getElementById(
-					instance.fromInputName
-				);
-				instance.toInputPicker = document.getElementById(
-					instance.toInputName
-				);
+				instance.fromInputPicker = A.one(`#${instance.fromInputName}`);
+				instance.toInputPicker = A.one(`#${instance.toInputName}`);
```

Comment:
No change needed, but as a note, it's best to use native javascript over AUI where we can. Better for getting used to not using AUI. Also when we want to remove AUI, it could be a smoother process when that time comes.